src/SUMMARY.md

```diff
@@ -41,9 +41,10 @@
 
 * [PHP](languages/php.md)
 * [Node.js](languages/nodejs.md)
-* [Python (beta)](languages/python.md)
-* [Ruby (beta)](languages/ruby.md)
-* [Go (beta)](languages/go.md)
+* [Python](languages/python.md)
+* [Ruby](languages/ruby.md)
+* [Go (beta))](languages/go.md)
+* [Java (alpha)](languages/java.md)
```

Comment:
What are the criteria for that?  I thought everything was beta until we had someone using it in production.  Do we know if there's someone using Python/Ruby in production?